main.js

```diff
@@ -1,5 +1,5 @@
 // Modules to control application life and create native browser window
-const {app, BrowserWindow} = require('electron')
+const {app, ipcMain, BrowserWindow} = require('electron')
 const path = require('path')
 
```

Comment:
would love to see `crashReporter` initialized here!

```
require('electron').crashReporter.start({ uploadToServer: false, submitURL: '' })
```

Reply:
Added in 544712d14681697e4c155e09ff509e7461216e7a